CMakePresets.json

```
@@ -6,56 +6,61 @@
     "patch": 0
   },
   "configurePresets": [
+    {
+      "name": "default",
+      "hidden": true,
+      "binaryDir": "${sourceDir}/build/cmake-build-${presetName}"
```

раньше не было `/build/`, го не будем саму директорию менять

```suggestion
      "binaryDir": "${sourceDir}/cmake-build-${presetName}"
```